azure/template.json

```diff
@@ -343,6 +343,29 @@
                 "app-service-plan"
             ]
         },
+        {
+            "name": "app-service-logs",
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2017-05-10",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'), 'app-service-logs.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "appServiceName": {
+                        "value": "[variables('appServiceName')]"
+                    },
+                    "httpLoggingEnabled": {
```

Comment:
Remember to add parameters for the additional logging options available for Windows apps as per https://github.com/DFE-Digital/manage-courses-ui/blob/58f6adefb2717ca496f8509d6fab24f0604108d4/azure/template.json#L280-L291

Reply:
Added additional logging options for windows apps